[2022-11-28T11:45:15.489+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_try_oop.transform manual__2022-11-28T11:44:44.008278+00:00 [queued]>
[2022-11-28T11:45:15.561+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_try_oop.transform manual__2022-11-28T11:44:44.008278+00:00 [queued]>
[2022-11-28T11:45:15.562+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-28T11:45:15.562+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2022-11-28T11:45:15.562+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-28T11:45:15.675+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): transform> on 2022-11-28 11:44:44.008278+00:00
[2022-11-28T11:45:15.726+0000] {standard_task_runner.py:55} INFO - Started process 1509 to run task
[2022-11-28T11:45:15.771+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_try_oop', 'transform', 'manual__2022-11-28T11:44:44.008278+00:00', '--job-id', '576', '--raw', '--subdir', 'DAGS_FOLDER/dag_try_oop.py', '--cfg-path', '/tmp/tmpg3xi_b7a']
[2022-11-28T11:45:15.783+0000] {standard_task_runner.py:83} INFO - Job 576: Subtask transform
[2022-11-28T11:45:16.232+0000] {task_command.py:376} INFO - Running <TaskInstance: dag_try_oop.transform manual__2022-11-28T11:44:44.008278+00:00 [running]> on host 32dc5aa7602c
[2022-11-28T11:45:16.708+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag_try_oop
AIRFLOW_CTX_TASK_ID=transform
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T11:44:44.008278+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-28T11:44:44.008278+00:00
[2022-11-28T11:45:17.019+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-11-28T11:45:17.088+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dag_try_oop, task_id=transform, execution_date=20221128T114444, start_date=20221128T114515, end_date=20221128T114517
[2022-11-28T11:45:17.272+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-28T11:45:17.398+0000] {taskinstance.py:2623} INFO - 1 downstream tasks scheduled from follow-on schedule check
